ironfist/src/cpp/resource/resourceManager.cpp

```diff
@@ -154,6 +154,16 @@ font* resourceManager::GetFont(char* filename) {
 
 #ifndef EDITOR
 
+sample* resourceManager::GetSample(const std::string &filename) {
```

Comment:
Uhhh.....what?

This was a comment on your previous implementation that didn't use c_str.